code/__defines/mobs.dm

```diff
@@ -258,9 +258,9 @@
 #define SPECIES_ALIEN            "Humanoid"
 #define SPECIES_GOLEM            "Golem"
 
-#define BODYTYPE_HUMANOID        "Humanoid Body"
-#define BODYTYPE_OTHER           "Alien Body"
-#define BODYTYPE_MONKEY          "Small Humanoid Body"
+#define BODYTYPE_HUMANOID        "humanoid body"
```

Comment:
wouldn't this break all the shit because icon states are named with Capitalized Version in dmis?

Reply:
They're named in lowertext in dmis.